tests/helpers.py

```diff
@@ -23,26 +22,3 @@
 WHEEL_FIXTURE = os.path.join(FIXTURES_DIR, "twine-1.5.0-py2.py3-none-any.whl")
 NEW_SDIST_FIXTURE = os.path.join(FIXTURES_DIR, "twine-1.6.5.tar.gz")
 NEW_WHEEL_FIXTURE = os.path.join(FIXTURES_DIR, "twine-1.6.5-py2.py3-none-any.whl")
-
-
-@contextlib.contextmanager
-def set_env(**environ):
```

Comment:
Any reason not to repurpose this to accept the monkeypatch provided to the test to do the loop here instead of in each individual test?

Reply:
No, not really. I went back and forth a couple of times between a function and the inline version. For the function I wanted to have the env vars definition be declared directly in the function arguments, but I really didn't like how black formats that, thus I decided for the inline version. It is a construct used just an handful of times and, counting the import statements, having the function almost makes the total number of lines of code larger. I can declare a function if you prefer.